content/es/guides/directory-structure/pages.md

```diff
@@ -0,0 +1,359 @@
+
+</base-alert>
+
+### loading
+
+Si esta propiedad tiene un valor de `false`, previene la página de ejecutar `this.$nuxt.$loading.finish()` automáticamente cuando entras a ella y `this.$nuxt.$loading.start()` cuando te vas. Esto te permite controlar el comportamiento de la página manualmente, como se muestra en [este ejemplo](https://nuxtjs.org/examples/custom-page-loading).
+
+```js{}[pages/index.vue]
+export default {
+  loading: false
+}
+```
+
+<base-alert type="info">
+
+Sólo aplica si `loading` está configurado también en el `nuxt.config.js`
+
+</base-alert>
+
+<base-alert type="next">
+
+Conoce más sobre esto en el capítulo de [Loading](/guides/features/loading).
+
+</base-alert>
+
+### transition
+
+La propiedad de transición define una transición específica para la página.
+
+```js{}[pages/index.vue]
+export default {
+  transition: 'fade'
+}
+```
+
+<base-alert type="next">
+
+Conoce más sobre transiciones en el capítulo de [Transiciones](/guides/features/transitions).
+
+</base-alert>
+
+### scrollToTop
+
+La propiedad de `scrollToTop` te permite decirle a Nuxt.js que se desplpaze hacia el tope de la página antes de que se muestre. Nuxt.js desplaza la página hacia el tope por defecto, pero en rutas internas, Nuxt.js mantiene la posición de desplazamiento (scroll position). Si deseas que Nuxt se desplaze al tope cuando muestre las rutas internas, configura `scrollToTop` a cierto (`true`).
+
+```js{}[pages/index.vue]
+export default {
+  scrollToTop: true
+}
+```
+
+También puedes configurar `scrollToTop` a falso manualmente en las rutas externas (parent routes), si deseas.
+
+Si quieres sobre escribir el comportamiento de desplazamiento (scroll) de Nuxt.js, puedes navegar a [opción de scrollBehavior](/guides/configuration-glossary/configuration-router#scrollbehavior)
+
+### middleware
+
+Puedes definir `middleware` para cada página. El `middleware` se ejecuta antes de que una página se muestre.
+
+```js{}[pages/index.vue]
+export default {
+  middleware: 'auth'
+}
+```
+
+<base-alert type="next">
+
+Conoce más sobre este tema en el capítulo de [Middleware](/guides/directory-structure/middleware).
+
+</base-alert>
+
+### La propiedad de watchQuery
+
+Esta propiedad se usa para configurar un observador para los `query strings`. Si el string definido cambia, todos los metodos de los componentes (asyncData, fetch, validate, layout, ...) se ejecutarán. El observador está disabilitado por defacto para mejorar la rapidéz de las páginas
+
+```js{}[pages/index.vue]
+export default {
+  watchQuery: ['page']
+}
+```
+
+<base-alert type="info">
+
+Si quieres configurar un observador (watcher) para todos los `query strings`, establece el valor de `watchQuery` a que sea cierto (`true`).
+
+</base-alert>
+
+```js{}[pages/index.vue]
+export default {
+  watchQuery: true
+}
+```
+
+También puedes usar la función de `watchQuery(newQuery, oldQuery)` para tener ovservadores más específicos.
```

Comment:
The word **_ovservadores_** should be _**observadores**_ .

Reply:
- [ ] fix